Mathlib/Analysis/Seminorm.lean

```diff
@@ -4,7 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Jean Lo, Yaël Dillies, Moritz Doll
 
 ! This file was ported from Lean 3 source module analysis.seminorm
-! leanprover-community/mathlib commit 7ebf83ed9c262adbf983ef64d5e8c2ae94b625f4
+! leanprover-community/mathlib commit 09079525fd01b3dda35e96adaa08d2f943e1648c
```

Comment:
It's usually best to merge in the latest `start_port.sh` output rather than blindly updating the SHA; that keeps it possible to diff against mathport

Reply:
Albeit there was only a [one-line change](https://github.com/leanprover-community/mathlib/commit/09079525fd01b3dda35e96adaa08d2f943e1648c#diff-d79b1546f965b69cf8e9270215158a6bb987ff43c84cfe5a3fa22e01fa7bd537) in mathlib3 with respect to this file.